app/js/header5.js

```diff
@@ -0,0 +1,44 @@
+var touches = document.querySelectorAll('.fifth__touch');
```

Comment:
if you want to add js function on some event to the button or whatever, please, create separate class with `js-` prefix.
For example:
```html
<button class="my-button js-hide-button">my button</button>
```
```javascript
var button = document.querySelector('.js-hide-button');

button.addEventListener('click', function(){
  this.classList.add('hidden')
})
```
So now you can see, that this button has some js functions on it. And you can rename `my button` class in any moment without editing all the js files, where you used this class to add some eventListeneres.

So, please add class with `js-` prefix to arrow buttons, you can name it like `js-swipe-prev` and `js-swipe-next` and you this class inside js functions

Reply:
I corrected, but please review this again, because I don't have feel like all good with prefix.
Thanks.